### DISCUSSION TITLE
Seizure Forecasting

### DISCUSSION THREAD

**Discussion by jones139** [OpenSeizureDetector/Android_Pebble_SD#72] [06_Epilepsy_Research]
Another one that Eduardo has found:  https://onlinelibrary.wiley.com/doi/full/10.1111/epi.16719 "Machine learning from wristband sensor data for wearable, noninvasive seizure forecasting"

I haven't had chance to have a good look at it yet, but might be something we can think about for OpenSeizureDetector?

**Comment by chrisroge** [comment]
I was just thinking we might be able to feed the raw data from the app, say for the last 24 hours, to OpenAI API, along with our own seizure logs and ask for an analysis from ChatGPT.  I have done some tinkering with the API and it does a nice job of responding in whatever format you specify.  In this case we could perhaps ask it to respond in a specific JSON or XML structure that the app could read.  If someone could give me a head start on how to get the raw data I will see what I can put together in the OpenAI playground.

**Comment by jones139** [comment]
It could be worth a try - I am a bit of a sceptic about some of these generic AI systems though.
We have a database of anonymised seizure (and non-seizure) data that I am currently using to train a neural network to do the seizure detection (and hopefully show less false alarms).   
You can have access to the data as long as you don't mind complying with the licence described here: https://github.com/OpenSeizureDetector/OpenSeizureDatabase  (which basically says you have to publish your results so we have the opportunity to incorporate your findings back into OpenSeizureDetector).
